docs/release-notes/breaking-changes.md

```diff
@@ -16,9 +16,9 @@ Breaking changes can impact your Elastic applications, potentially disrupting no
 
 ## 3.0.0 [elastic-cloud-kubernetes-300-breaking-changes]
 
-::::{dropdown} Title of breaking change
-Description of the breaking change.
-For more information, check [PR #](PR link).
-**Impact**<br> Impact of the breaking change.
-**Action**<br> Steps for mitigating deprecation impact.
+::::{dropdown} Enterprise search no longer available with 9.0.0
+The standlaone Enterprise search, App Search and Workplace Search products remain available in maintenance mode and are no longer recommended for new search experiences. We recommend transitioning to our actively developed Elastic Stack tools to build new semantic and AI powered search experiences.There will be no stand alone Enterprise search 9.x image to update to.
+For more information, check [Migrating to 9.x from Enterprise Search 8.x versions](https://www.elastic.co/guide/en/enterprise-search/8.18/upgrading-to-9-x.html).
+**Impact**<br> Upgrade to 9.0.0 will be blocked due to stand alone Enterprise search 
+**Action**<br> check [Migrating to 9.x from Enterprise Search 8.x versions](https://www.elastic.co/guide/en/enterprise-search/8.18/upgrading-to-9-x.html).
```

Comment:
<img width="778" alt="Screenshot 2025-03-26 at 14 21 20" src="https://github.com/user-attachments/assets/6141e2db-904c-4bfb-a36a-dbf19d6d1bce" />

This does not render very nicely. Needs at least a few more line breaks. Also I am not sure we should link twice in the same paragraph to the same document.

Reply:
```suggestion
**Action**<br> Migrate away from Enterprise Search following [this guide](https://www.elastic.co/guide/en/enterprise-search/8.18/upgrading-to-9-x.html). Only once the standalone Enterprise Search resources have been deleted attempt the upgrade of the Elastic stack to version 9.0.0.
```